docs/README.md

```diff
@@ -1,16 +1,18 @@
-# AWS SDK for Java 2.0 Docs
+# AWS SDK for Java 2.x Documentation
 
-## 1.11 to 2.0 Changelog
-- [Launch Changelog](LaunchChangelog.md)
+## SDK Usage
 
-## New Contributions
-- [Getting Started](GettingStarted.md)
```

Comment:
No link anywhere to this anymore?

Reply:
The GettingStarted.md is linked to by Contributing.md, which is already linked. It was weird to direct people to "getting started" when "contributing" seemed like the right channel.